packages/ng/styles/components/cdk/_overlay.scss

```diff
@@ -30,6 +30,10 @@
 	position: absolute;
 	pointer-events: auto;
 	z-index: 1000;
+
+	> * {
+		flex-grow: 1;
+	}
```

Comment:
Not sure about this but shouldn't this be in a dedicated PR?

Reply:
This one is due to a change in cdk that adds flex to the default overlay container, this SCSS is here as a workaround for this.